.gitignore

```diff
@@ -13,3 +13,10 @@ run
 .classpath
 .project
 .settings/
+/bin/
```

Comment:
`/bin/`? What is this supposed to be? Did you mean to write `./bin` instead/

Reply:
I only added to .gitignore starting from src/generated, the rest was already there.

Reply:
that is unfortunately not true, here is a screenshot of what the gitignore file looks like in the `main` branch 
<img width="687" alt="image" src="https://user-images.githubusercontent.com/44499264/157149492-7b010fda-2de5-4e48-a83b-cd5d996be6d4.png">

Reply:
@jliyazhu12 if you never meant to add `bin` in there,  I suppose you could just scrap that line entirely.

Reply:
I honestly have no idea how that was added, I'll remove it.